BuildTools/build-1.18+.sh

```diff
@@ -8,7 +8,7 @@ MAVEN_DIR="$HOME/.m2"
 SPIGOT="${MAVEN_DIR}/repository/org/spigotmc/spigot"
 
 #Versions
-array=("1.18.1")
+array=("1.18.2", "1.18.1")
```

Comment:
This should be like this, bash is silly with it's arrays :)
```suggestion
array=("1.18.2" "1.18.1")
```

Reply:
Oh, good to know :D Thats why this isn't worked for me :'D